ansible/vars/topo_m0.yml

```diff
@@ -87,23 +87,23 @@ configuration_properties:
     dut_asn: 65100
     dut_type: MgmtToRRouter
     podset_number: 30
-    tor_number: 16
-    tor_subnet_number: 2
```

Comment:
Seems like we have other place using these field, such as: https://github.com/sonic-net/sonic-mgmt/blob/60886753bd4e4ddb582438c9a0f09711699e792d/tests/vrf/vrf_fib.j2

Can you help confirm?